content/200-concepts/050-overview/300-prisma-in-your-stack/03-is-prisma-an-orm.mdx

```diff
@@ -1,215 +1,157 @@
 ---
 title: 'Is Prisma an ORM?'
 metaTitle: 'Is Prisma an ORM?'
-metaDescription: 'Learn why Prisma is not an ORM. It shares similar goals with ORMs and wants to make working with databases easy, but it does not map classes to tables as ORMs do.'
+metaDescription: 'Learn about how Prisma implements the Data Mapper ORM pattern and how it achieves the same goal as traditional ORMs without requiring you to map classes to tables as traditional ORMs do.'
 ---
 
-<TopBlock>
+## Overview
 
-To answer the question briefly: _No, Prisma is not an ORM_.
+To answer the question briefly: _Yes, Prisma is a new kind of ORM that fundamentally differs from traditional ORMs and doesn't suffer from many of the problems commonly associated with these_.
 
-The main goal of Prisma is to make working with databases easier for application developers. In that sense, it shares the same goals with ORMs, but takes a fundamentally different approach.
+Traditional ORMs provide an object-oriented way for working with relational databases by mapping tables to _model classes_ in your programming language. This approach leads to many problems that are caused by the [object-relational impedance mismatch](https://en.wikipedia.org/wiki/Object%E2%80%93relational_impedance_mismatch).
 
-ORMs are libraries that map tables in your database to classes in your programming language. Prisma, on the other hand, is a database toolkit. The toolkit includes Prisma Client, which is an auto-generated query builder that exposes queries which are _tailored_ to your models. All Prisma Client queries return plain old JavaScript objects.
+Prisma works fundamentally different compared to that. With Prisma, you define your models in the declarative [Prisma schema](../../reference/tools-and-interfaces/prisma-schema/prisma-schema-file) which serves as the single source of truth for your database schema and the models in your programming language. In your application code, you can then use Prisma Client to read and write data in your database without the overhead of managing complex model instances. This makes the process of querying data a lot more natural as well as more predictable since Prisma Client always returns plain JavaScript objects.
```

Comment:
Thought: Mention type safety here somehow?

Reply:
Updated the line to 

> In your application code, you can then use Prisma Client to read and write data in your database **in a type-safe manner** without the overhead of managing complex model instances.